packages/mgt-components/src/components/mgt-person/mgt-person.scss

```diff
@@ -32,6 +32,8 @@ $line3-text-transform: var(--line3-text-transform, #{$text-transform});
 
 $details-spacing: --details-spacing;
 
+$avatar-direction: --avatar-direction;
```

Comment:
`--person-flex-direction` seems more appropriate. Make sure to also document this as a `cssprop` in the person class

Reply:
Great idea, updated the names in all associated locations:

c4b7760